Add new cgroup metrics metadata (#20564)
DataDog/integrations-core: container/README.md

### Configuration

> The `container` check exposes specific configuration settings to emit extended memory metrics in addition to standard set. To customize common fields or to force the activation of the `container` check, follow these steps:

Reviewer: ```suggestion
The `container` check exposes specific configuration settings to emit extended memory metrics in addition to the standard set. To customize common fields or to force the activation of the `container` check, follow these steps:
```

See [metadata.csv][3] for a list of metrics provided by this integration.

> If you need extended container memory metrics `extended_memory_metrics` has to be set to `true`. It will enable following metrics: active_anon, inactive_anon, active_file, inactive_file, unevictable, shmem, file_mapped, file_dirty, file_writeback, page_tables, refault_anon, refault_file

Reviewer: ```suggestion
If you need extended container memory metrics, `extended_memory_metrics` has to be set to `true`. It enables the following metrics: `active_anon`, `inactive_anon`, `active_file`, `inactive_file`, `unevictable`, `shmem`, `file_mapped`, `file_dirty`, `file_writeback`, `page_tables`, `refault_anon`, and `refault_file`.
```
